altair-spring-boot-autoconfigure/src/main/java/com/oembedler/moon/altair/boot/AltairController.java

```diff
@@ -52,6 +52,9 @@
     @Value("${altair.cdn.version:2.1.4}")
     private String altairCdnVersion;
 
+    @Value("${altair.google-analytics.enabled:true}")
```

Comment:
Could you define this property in https://github.com/graphql-java-kickstart/graphql-spring-boot/blob/master/altair-spring-boot-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json as well so it's available in intellisense?

Reply:
Yes, good suggestion. Just added it in the latest commit.